src/fastcs/logging/__init__.py

```
@@ -102,3 +104,13 @@ def intercept_std_logger(logger_name: str):
     std_logger.handlers = [handler]
     std_logger.propagate = False
+
+
+# For sphinx autodoc to find things import from private modules
+__all__ = [
+    "logger",
+    "bind_logger",
+    "configure_logging",
+    "intercept_std_logger",
+    "GraylogEndpoint",
+]
```

Ruff (0.15.5)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)